cjwmodule/i18n/funcs.py

```diff
@@ -34,4 +34,32 @@ def trans(id: str, default: str, args: Dict[str, Any] = {}) -> I18nMessage:
     :param args: Values to interpolate into the message. (e.g.,
                  ``{"column": "A"}```
     """
-    return I18nMessage(id, args)
+    return I18nMessage(id, args, "module")
+
+
+def trans_cjwmodule(id: str, default: str, args: Dict[str, Any] = {}) -> I18nMessage:
```

Comment:
Please make the `trans_cjwmodule` a _private_ function (preceded by `_`). Module authors (`cjwmodule`'s only users) can't use it, so we shouldn't expose it to them.